android/cloudbuild.yaml

```
@@ -1,24 +1,27 @@
 steps:
-- name: 'gcr.io/cloud-builders/gsutil'
-  entrypoint: "bash"
-  args: 
-  - '-c'
-  - |
-    gsutil rsync -d -r "${_LICENSES_BUCKET}" licenses || exit 0
-  id: copy-licenses
+  
 - name: 'gcr.io/cloud-builders/docker'
   args:
   - 'build'
-  - '--tag=gcr.io/$PROJECT_ID/android-builder'
-  - "--build-arg"
-  - "ANDROID_SDK_LICENSE=${_ANDROID_SDK_LICENSE}"
+  - '--tag=gcr.io/$PROJECT_ID/android:base'
+  - '--file=Dockerfile-base'
+  - '.'
+
+- name: 'gcr.io/cloud-builders/docker'
+  args:
+  - 'build'
+  - '--tag=gcr.io/$PROJECT_ID/android:${_ANDROID_VERSION}'
+  - '--build-arg'
+  - 'android_version=${_ANDROID_VERSION}'
+  - '--build-arg'
+  - 'project_id=$PROJECT_ID'
+  - '--file=Dockerfile-sdk'
   - '.'
   - '--cache-from'
-  - 'gcr.io/$PROJECT_ID/android-sdk-base'
-  waitFor: 
-  - copy-licenses
-images: ['gcr.io/$PROJECT_ID/android-builder']
+  - 'gcr.io/$PROJECT_ID/android:base'
+
+images: 
```

Nit: remove trailing whitespace.
```suggestion
images:
```